api/tests/core/offers/test_validation.py

```diff
@@ -739,10 +740,31 @@ def test_allow_creation_with_inactive_ean(self):
 
 @pytest.mark.usefixtures("db_session")
 class CheckBookingLimitDatetimeTest:
-    def test_check_booking_limit_datetime_should_raise_because_booking_limit_is_one_hour_after(self):
-        venue = offerers_factories.VenueFactory(departementCode=71)
-        offer = offers_factories.OfferFactory(venueId=venue.id)
-        stock = offers_factories.StockFactory(offerId=offer.id)
+    @pytest.mark.parametrize(
+        "stock_factory_function, offer_factory_function, venue_factory_fonction",
+        [
+            (
+                educational_factories.CollectiveStockFactory,
+                educational_factories.CollectiveOfferFactory,
+                offerers_factories.VenueFactory,
+            ),
+            (
+                offers_factories.StockFactory,
+                offers_factories.DigitalOfferFactory,
+                offerers_factories.VirtualVenueFactory,
+            ),
+            (offers_factories.StockFactory, offers_factories.OfferFactory, offerers_factories.VenueFactory),
+        ],
+    )
+    def test_check_booking_limit_datetime_should_raise_because_booking_limit_is_one_hour_after(
+        self, stock_factory_function, offer_factory_function, venue_factory_fonction
```

Comment:
```suggestion
        self, stock_factory, offer_factory, venue_factory
```
Pas besoin de préciser le `_function`, l'absence de majuscule me semble suffisante.
Je ferai bien même:
```suggestion
        self, StockFactory, OfferFactory, VenueFactory
```
mais ça va surement faire raler pylint/mypy.